### ISSUE TITLE
What happens when someone in the group changes the group currency?

### ISSUE THREAD

**Issue by mmbotelho** [okTurtles/group-income#784]
### Problem

As @dotmacro pointed out in the "Group Settings" page on figma:

> If the default currency is changed, what happens to the mincome? That is, does it go from $100 -> €90 (equivalent value, different currency, different number of units) or from $100 -> €100 (different value, different currency, same number of units)?

It is not clear ATM what should happen when someone changes the currency of a group.

### Solution
Figure out how this flow should go. Here are some possible solutions:
- Change the currency of the group immediately and convert all values to the new currency, for everyone in the group (probably a bad idea);
- Change the currency from the group level and change it to a personal level, where each user can see the original currency AND their prefered currency - this is good for international groups, or for when the group currency is a crypto like Bitcoin (because it's so volatile, the value is only useful when placed next to a more stable fiat);
- If we do leave the currency at a group level, and a user changes it, we can ask if they want to convert all the values to the new currency, leave them as they are but just change the currency or just change the currency for the user.

**Comment by dotmacro** [comment]
> Change the currency of the group immediately and convert all values to the new currency, for everyone in the group (probably a bad idea);

Agreed. We'd need to figure out where to pull exchange rates from, and how often. This might be okay for changing between _relatively_ stable currencies like USD & EUR, but would not work well with anything volatile like VEF or BTC.

> Change the currency from the group level and change it to a personal level, where each user can see the original currency AND their preferred currency

This seems like a good option for the user, but we still need to decide where to pull exchange rates from, and how often. 

> If we do leave the currency at a group level, and a user changes it, we can ask if they want to convert all the values to the new currency, leave them as they are but just change the currency...

We could change the mincome proposal modal to be input + select, so the user can choose to change the mincome unit quantity, the mincome currency, or both. I think this is the best option, for both the users and the developers. (But it doesn't address users having a different preferred currency.)

**Comment by sandrina-p** [comment]
> We could change the mincome proposal modal to be input + select, so the user can choose to change the mincome unit quantity, the mincome currency, or both.
+1 for this option too.

> But it doesn't address users having a different preferred currency. 
> We still need to decide where to pull exchange rates from, and how often.

I have a suggestion: Next to the group income, add an icon (or the icon itself is clickable) and the user clicks there and we do a request to get the exchange rate at that precise moment.

**Comment by dotmacro** [comment]
> Next to the group income, add an icon (or the icon itself is clickable) and the user clicks there and we do a request to get the exchange rate at that precise moment.

Would this be on the mincome proposal modal (e.g. to help the user propose a change from 100EUR to 100EUR of BTC) or on the Dashboard (so the user can quickly see that 100USD is 90.72EUR), or both?

**Comment by taoeffect** [comment]
In the Mincome proposal, make it possible to change the currency using a dropdown. Currently the currency is unselectable. Make it a dropdown menu. When the currency is changed, use an API to convert the amount.

E.g. currently we have 100 USD. Dropdown clicked to change USD to BTC. Conversion done to change 100 USD to BTC and the 100 gets changed to `100 * USD_TO_BTC_RATE`.

**Comment by sandrina-p** [comment]
I like @taoeffect suggestion ☝️I think we should also remove "Default currency" input from Group Settings page.
